climakitae/explore/vulnerability.py

```diff
@@ -96,6 +137,10 @@ def check_and_set_params(
             "Passed in historical data method must be 'Historical Climate' or 'Historical Reconstruction'"
         )
 
+    # Validating warming level inputs
+    if warming_level not in ["1.5", "2.0", "3.0", "4.0"]:
+        raise ValueError("Passed in warming level must be '1.5', '2.0', '3.0', '4.0'")
+
     # Validating downscaling method
     downscaling_methods = ["Dynamical", "Statistical", "Dynamical+Statistical"]
```

Comment:
Remove "Dynamical+Statistical" as an option here. We don't want to advocate for users (at this point in time) mixing and matching data methods